app/templates/components/widgets/forms/time-picker.hbs

```diff
@@ -1,4 +1,4 @@
 {{#if icon}}
   <i class="time icon"></i>
 {{/if}}
-{{input type='text' value=value placeholder=placeholder name=name}}
+{{input type='text' value=value placeholder=placeholder id=inputId name=name}}
```

Comment:
@geekyd As mentioned in the issue : 
* it is a form element which requires validation without the id, it can't be validated, validation was broken in this case
* how are we supposed to submit the data to the server from a field if it lacks an identifier
* one trivial reason, id was passed while rendering the widget, and was supposed to be used.